### DISCUSSION TITLE
Need more recent SNAP version of zigbee2mqtt

### DISCUSSION THREAD

**Discussion by paulr7654** [community/community#109620] [Packages]
### Select Topic Area

Question

### Body

Current version is 1.30.1
When will 1.33.1 or more recent be available.
I have a sonoff trv which is not supported by 1.30.1 but is evidently supported by 1.33.1 (and higher)
Snap is a convenient way of accessing zigbee2mqtt- are you still releasing snaps. I hope so
With all good wishes 
Paul Rumph

**Comment by ettaboyle** [comment]
Hi @paulr7654,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's repository](https://github.com/zigbee2mqtt). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

In our [Community Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md), we ask that members contribute in a positive and constructive way. This type of post is considered "off topic" for our Community Discussions. Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding. 

Best of luck!
